### PR TITLE
Development

### LINKED ISSUE #31: SOUVENIRS - Create Souvenir

### LINKED ISSUE #32: STAFF - Print Create Form
printStaffForm

### LINKED ISSUE #33: SOUVENIRS - Create Form Builder

### LINKED ISSUE #34: STAFF - Create Form Builder
createStaffForm

### LINKED ISSUE #38: SOUVENIRS - Print Create Form

### LINKED ISSUE #68: FOOD - Create Food

### PR DESCRIPTION
**Pull request by jrobinson0529**
## Description
Added form buttons to open form for each module.
Added ability to create
- Food cards
- Souvenir cards
- Show cards
- Staff cards

Deployed site - https://renfaire.netlify.app/

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: INSTRUCTIONSmd [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Renaissance Fair Dashboard
```
Comment (drteresavasquez) [inline INSTRUCTIONSmd]: I think this file name needs to have a dot in it

File: src/javascripts/components/buildSouvenirs.js [hunk 2]
```diff
@@ -1,15 +1,16 @@
 const buildSouvenirs = (array) => {
-  document.querySelector('#content-container').innerHTML = '';
+  console.warn(array);
```
Comment (drteresavasquez) [inline src/javascripts/components/buildSouvenirs.js]: Get rid of the console warn

File: src/styles/main.scss [hunk 3]
```diff
@@ -1,12 +1,20 @@
 @import "~bootstrap/scss/bootstrap";
 @import "~@fortawesome/fontawesome-free/css/all.min.css";
+@import url('https://fonts.googleapis.com/css2?family=Alike&display=swap');
+@import './staff.scss';
+
+// #app {
```
Comment (drteresavasquez) [inline src/styles/main.scss]: remove commented out code

**Review by drteresavasquez — CHANGES_REQUESTED** [review]
Question. On the deployed app, when I click Souvenirs the title changes but the cards do not and when I click Food Shoppes nothing happens. Is that expected behavior?

File: INSTRUCTIONSmd (hunk 1 above)
Comment (hmparsons-sde) [reply INSTRUCTIONSmd]: You're right! Good catch!

Comment (jrobinson0529) [reply INSTRUCTIONSmd]: Oh we haven't incorporated the navlinks in yet. The links on the main card buttons are the same.

Comment (jrobinson0529) [reply INSTRUCTIONSmd]: We can make those links functional for consistency in the meantime!

**Review by drteresavasquez — APPROVED** [review]
👍🏼

**Pull request merged**